tests/test_instance_settings_handler.cpp

```diff
@@ -82,7 +82,9 @@ struct TestInstanceSettingsHandler : public Test
     std::unordered_map<std::string, mp::VirtualMachine::ShPtr> deleted_vms;
     std::unordered_set<std::string> preparing_vms;
     bool fake_persister_called = false;
-    inline static constexpr auto properties = std::array{"cpus", "disk", "memory"};
+    inline static constexpr auto numeric_properties = std::array{"cpus", "disk", "memory"};
+    inline static constexpr auto boolean_properties = std::array{"bridged"};
+    inline static constexpr auto properties = std::array{"cpus", "disk", "memory", "bridged"};
 };
```

Comment:
I know it stems from the original code, but maybe change the copy initialization to direct initialization to make the code cleaner. 
```
    inline static constexpr std::array numeric_properties{"cpus", "disk", "memory"};
    inline static constexpr std::array boolean_properties{"bridged"};
    inline static constexpr std::array properties{"cpus", "disk", "memory", "bridged"};
```

Reply:
Right, looks better that way. Changed.